pom.xml

```diff
@@ -67,6 +67,48 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19.1</version> 
```

Comment:
The POM is definitely merged correctly, take a look here https://github.com/jenkinsci/accurev-plugin/blob/development/pom.xml